Blog post: A new kind of Open Source Event

gr2m opened the pull request:
Based on draft at https://docs.google.com/document/d/1s4x2XsWjU729luQCwkJ2Dhne4PM8gLPXZMaHkBD9L8Y/edit?pref=2&pli=1

Preview: https://github.com/hoodiehq/hood.ie/blob/blog-post-a-new-kind-of-open-source-event/_posts/2016-03-15-a-new-kind-of-open-source-event.md

---

closes https://github.com/hoodiehq/editorial/issues/78

gr2m committed:
Blog Post: A new kind of Open Source Event

jennwrites committed:
Blog Post: A new Kind of Open Sou… (Jenn’s tweaks)

jennwrites commented on _posts/2016-03-15-a-new-kind-of-open-source-event.md:
```diff
@@ -0,0 +1,72 @@
+
+Thanks to Alex, Jan & Jenn for their reviews!
+
+The [online Word editor](http://wordhtml.com/ "Online Word Editor") will help you compose easily documents for the web.
```
What's this link?

gr2m replied:
LOL I used this editor in the process from Google Doc -> Markdow. What a scam to their ad in there, I totally missed it. Thanks for catching it :)

jennwrites commented:
Great job on this, I had no idea we were doing these events! Exciting stuff :smile: +1 to publish

gr2m commented:
> I had no idea we were doing these events

It kinda randomly happend :) Thanks for the review, I’ve removed the weird link and fixed up the commits. How is the process now, can we publish this now? Do we need another review? Is there a schedule when this post should go live?

jennwrites commented:
I think it best that each post gets reviewed by 2 people, as a rule.

As far as a schedule goes, if you want to get it out tonight, I’m okay with pushing this out there. Otherwise if it can wait until tomorrow, (which gives more time for review) we can do that too. 

-- 
Jenn Turner
I ❤️ great email at jennwrit.es/newsletters 

On March 15, 2016 at 3:51:28 PM, Gregor Martynus (notifications@github.com) wrote:

I had no idea we were doing these events

It kinda randomly happend :) Thanks for the review, I’ve removed the weird link and fixed up the commits. How is the process now, can we publish this now? Do we need another review? Is there a schedule when this post should go live?

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub

gr2m commented:
okay thanks :) Let’s wait for a second review

HipsterBrown commented:
Very awesome plan and event! I really like the reserved starter issues and openness. I'm looking forward to the results. :+1: LGTM!

gr2m commented:
@renrutnnej we have this integration with https://lgtm.co/, which requires two people from the maintainers team to comment with LGTM before it allows us to merge something. I did this just now so we have two, just wanted to let you know for the future :)
